jib-core/src/integration-test/java/com/google/cloud/tools/jib/api/ReproducibleImageTest.java

```diff
@@ -157,6 +157,7 @@ public void testAllFileAndDirectories() throws IOException {
             Assert.assertTrue(layerEntry.isFile() || layerEntry.isDirectory()));
   }
 
+  @SuppressWarnings("JdkObsolete") // use of Date from TarArchiveEntry
```

Comment:
is this unavoidable?

Reply:
Yeah, even the latest Apache Commons Compress returns `Date` when calling [`TarArchiveEntry.getLastModifiedDate()`](https://commons.apache.org/proper/commons-compress/javadocs/api-release/org/apache/commons/compress/archivers/tar/TarArchiveEntry.html#getLastModifiedDate--) or [`TarArchiveEntry.getModDate()`](https://commons.apache.org/proper/commons-compress/javadocs/api-release/org/apache/commons/compress/archivers/tar/TarArchiveEntry.html#getModTime--). They are the only ways to get the file modification time.